File: app/src/main/java/org/rm3l/router_companion/exceptions/StorageException.java
package org.rm3l.router_companion.exceptions;

import android.support.annotation.Nullable;

/**
 * Created by rm3l on 13/12/15.
 */
public class StorageException extends DDWRTCompanionException {

    public StorageException() {
    }

    public StorageException(@Nullable String detailMessage) {
        super(detailMessage);
    }

    public StorageException(@Nullable String detailMessage, @Nullable Throwable throwable) {
        super(detailMessage, throwable);
    }

    public StorageException(@Nullable Throwable throwable) {
        super(throwable);
    }
}

File: app/src/main/java/org/rm3l/router_companion/exceptions/StorageException.kt
package org.rm3l.router_companion.exceptions

/**
 * Created by rm3l on 13/12/15.
 */
class StorageException : DDWRTCompanionException {

  constructor() 

  constructor(detailMessage: String?) : super(detailMessage) 

  constructor(detailMessage: String?, throwable: Throwable?) : super(detailMessage, throwable) 

  constructor(throwable: Throwable?) : super(throwable) 
}
